### PR TITLE
Remove I386_ONLY support

### PR DESCRIPTION
**Pull request by richsalz**
Should we just remove I386 support altogether?
## Checklist
Thank you for your contribution. Please answer the following:

- [X] I acknowledge that I am authorized to submit this code under
the terms of the [Apache License](https://www.apache.org/licenses/LICENSE-2.0)
- [ ] All new public APIs are documented
- [ ] All new public functions have tests
- [ ] All new public functions have fuzzing tests

### CODE REVIEW

File: crypto/modes/gcm128.c [hunk 1]
```diff
@@ -337,7 +337,7 @@ void gcm_ghash_4bit(u64 Xi[2], const u128 Htable[16], const u8 *inp,
 # endif
 
 #if     (defined(GHASH_ASM) || defined(OPENSSL_CPUID_OBJ))
-# if    !defined(I386_ONLY) && \
+# if    \
         (defined(__i386)        || defined(__i386__)    || \
```
Comment (botovq) [inline crypto/modes/gcm128.c:341]: These two lines should be merged:
```
# if    (defined(__i386)        || defined(__i386__)    || \
```

File: crypto/sha/sha512.c [hunk 2]
```diff
@@ -402,23 +402,12 @@ static const SHA_LONG64 K512[80] = {
...
-#     define PULL64(x) ({ const unsigned int *p=(const unsigned int *)(&(x));\
-                          unsigned int hi=p[0],lo=p[1];          \
-                                asm("xchgb %%ah,%%al;xchgb %%dh,%%dl;"\
-                                    "roll $16,%%eax; roll $16,%%edx; "\
-                                    "xchgb %%ah,%%al;xchgb %%dh,%%dl;"\
-                                : "=a"(lo),"=d"(hi)             \
-                                : "0"(lo),"1"(hi) : "cc");      \
-                                ((SHA_LONG64)hi)<<32|lo;        })
-#    else
 #     define PULL64(x) ({ const unsigned int *p=(const unsigned int *)(&(x));\
```
Comment (botovq) [inline crypto/sha/sha512.c:405]: Wrong indent for the #define

**Review by botovq — APPROVED** [review]
x86asm.pl mentions `$i386only` in a comment, which should probably be corrected to `$x86only`.

File: crypto/modes/gcm128.c (hunk 1 above)
Comment (richsalz) [reply crypto/modes/gcm128.c:341]: I did a general cleanup/tweak of the spacing and extraneous parens.

**Comment by richsalz** [comment]
@botovq addressed your concerns, please take a look if you get a chance.

**Review by botovq — APPROVED** [review]

**Pull request merged**